instruqt-tracks/terraform-cloud-aws-v2/02-terraform-cloud-setup/check-workstation

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -l
+set -e
+
+# Create /tmp/skip-check to disable this check
+if [ -f /tmp/skip-check ]; then
+    rm /tmp/skip-check
+    exit 0
+fi
+
+export ORG=$(grep organization /root/terraform-cloud/terraform.tfvars | cut -d '"' -f2)
+export WORKSPACE=$(grep workspace /root/terraform-cloud/terraform.tfvars | cut -d '"' -f2)
+
+agent variable set TF_ORG $ORG
+agent variable set TF_WORKSPACE $WORKSPACE
+
+cp /root/hashicat-aws/remote_backend.tf.example /root/hashicat-aws/remote_backend.tf
+
+cd /root/hashicat-aws
+sed -i "s/YOUR_ORGANIZATION/$ORG/g" remote_backend.tf
+sed -i "s/YOUR_WORKSPACE/$WORKSPACE/g" remote_backend.tf
+
+cd /root/terraform-api
+sed -i "s/YOUR_ORGANIZATION/$ORG/g" terraform.tfvars.example
+mv terraform.tfvars.example terraform.tfvars
+
+# Store the ORG in /root/.bashrc
```

Comment:
This seems moderately fragile, should we just add the following to `/root/.bashrc` to replace 26-30?
```
export ORG=$(agent variable get ORG)
export WORKSPACE=$(agent variable get WORKSPACE)
```

Reply:
Fair enough. Can optimize

Reply:
Even with Instruqt variables available, some things I like to still write persistently to a .sh file inside the track.  That way if I need to troubleshoot a bugged-out session, I can easily have the student dump the file and I can see what's been stored in it.
(Typically I store it in a file named `[participant-id]-[something].sh`, so if it later becomes important which track instance it was when talking to Instruqt, we'll already know from having the student dump the file.  And at track setup I `touch` that file and add a line to `/root/.bashrc` so it gets sourced in every shell.  I think I swiped that idea from one of Gilberto's tracks.)